src/dash-table/DataTable.js

```diff
@@ -537,6 +540,48 @@ export const propTypes = {
         })).isRequired
     })),
 
+    column_static_tooltip: PropTypes.objectOf(PropTypes.shape({
+        type: PropTypes.oneOf([
+            'text',
+            'markdown'
+        ]),
+        value: PropTypes.string
+    })),
+
+    column_conditional_tooltips: PropTypes.arrayOf(PropTypes.shape({
+        if: PropTypes.shape({
```

Comment:
Even if it works, makes me a bit uneasy to use a keyword `if` as a key. Perhaps `condition`?

Reply:
So this follows the same pattern that's been used elsewhere (e.g. dropdown and style conditions) -- there was a pretty long discussion involving @cldougl and @chriddyp over the use of 'if' vs 'condition' vs other things I don't quite remember at the top of my head. General feeling was that the possible confusion was offset by the clarity and terseness.

Reply:
> General feeling was that the possible confusion was offset by the clarity and terseness.

yeah that was it as I remember:
didn't want to use `if` but the overall syntax didn't look user friendly/readable when using `conditional` or more specific descriptors

Reply:
OK, if this is matching other usage in the ecosystem I can live with it :)